.gitignore

```diff
@@ -9,3 +9,4 @@ output/
 bazel-*
 _bazel_*
 node_net.md
+/src/lib/gps/build
```

Comment:
Questa modifica direi che non ha senso in generale, probabilmente era servita a te in locale ma la toglierei